platform/src/main/java/org/stellar/anchor/platform/service/PaymentOperationToEventListener.java

```diff
@@ -257,5 +257,13 @@ void handleSep6Transaction(ObservedPayment payment, JdbcSep6Transaction txn)
         payment.getTransactionHash(),
         payment.getAmount(),
         rpcConfig.getCustomMessages().getIncomingPaymentReceived());
+
+    Metrics.counter(
+            AnchorMetrics.SEP6_TRANSACTION_OBSERVED.toString(),
+            "status",
+            SepTransactionStatus.PENDING_ANCHOR.toString())
+        .increment();
```

Comment:
We have two metrics patterns. 

WDYT to use something like:
```suggestion
Counter sep6WithdrawalCounter =
      counter(
          MetricConstants.SEP6_TRANSACTION_OBSERVED
          MetricConstants.STATUS,
          MetricConstants.PENDING_ANCHOR);
sep6WithdrawalCounter.increment();
```

Reply:
I think we should probably differentiate tags and metrics either by type or by making them members of different classes.

Reply:
Do it in a separate PR?

Reply:
Yep